### PR TITLE
Support collect test results based on build number for pipeline type

### PR DESCRIPTION
**Pull request by ngocmanhha**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/java-libraries/job/test-results-aggregator-plugin-tests-zuul-el7/1/ : SUCCESS in 1m 44s
- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-oss-check/1/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/java-libraries/job/test-results-aggregator-plugin-tests-zuul-el7/3/ : SUCCESS in 1m 52s
- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-oss-check/3/ : SUCCESS in 6s

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+all: tarball
+	
+tarball:
+	git archive --format=tar HEAD | gzip > jenkins-in-house-plugins-test-results-aggregator-plugin.tar.gz
```
Comment (helobinvn) [inline Makefile:4]:
```suggestion
	git archive --format=tar HEAD | gzip > jenkins-in-house-plugins-test-results-aggregator.tar.gz
```

File: test-results-aggregator-plugin.spec [hunk 2]
```diff
@@ -0,0 +1,41 @@
+%global     plugin_name test-results-aggregator-plugin
```
Comment (helobinvn) [inline test-results-aggregator-plugin.spec:1]:
```suggestion
%global     plugin_name test-results-aggregator
```

**Comment by helobinvn** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-fossa-scanning-zuul/1/ : FAILURE in 3m 15s (non-voting)
- https://checklist.intgdc.com/job/java-libraries/job/test-results-aggregator-plugin-tests-zuul-el7/4/ : SUCCESS in 1m 48s
- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-merge-compliance-check/1/ : SUCCESS in 19s

**Pull request merged**